app/src/main/java/org/mozilla/focus/network/SocketTags.java

```diff
@@ -2,4 +2,5 @@
 
 public class SocketTags {
     public static final int SEARCH_SUGGESTION = 10000;
+    public static final int DOWNLOADS = 10001;
```

Comment:
NIT: Maybe DOWNLOAD_HEADER? since not every download will use this and the download itself will not be tagged with this.

Reply:
I prefer to have a generic tag instead of specific one, we may reuse this tag in other operations which download something from network.